04/index.html

```diff
@@ -1,30 +1,36 @@
 <!DOCTYPE html>
 <html lang="pl">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/global.css" />
+    <link rel="stylesheet" href="css/mobile.css" />
+    <link rel="stylesheet" media="(min-width: 700px)" href="css/desktop.css" />
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
-</head>
-<body>
-    
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
-            </li>
-            <li>
-                <a href="#">oferta</a>
-                <ul>
-                    <li><a href="">strony internetowe</a></li>
-                    <li><a href="">pozycjonowanie stron www</a></li>
-                </ul>
-            </li>
-            <li>
-                <a href="#">kontakt</a>
-            </li>
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <label  for="menu-switcher" class="hamburger">
+        </label>
+        <input type="checkbox" id="menu-switcher" />
+        <ul class="nav__ul nav__ul--visibility">
+          <li class="nav__li">
+            <a class="nav__a" href="#">start</a>
+          </li>
+          <li class="nav__li nav__li--active">
+            <a class="nav__a nav__a--offer" href="#">oferta</a>
+            <ul class="nav__ul nav__ul--secondary">
+              <li class="nav__li--secondary"><a class="nav__a nav__a--secondary" href="">strony internetowe</a></li>
```

Comment:
Jesli mamy modyfikator to też element powinien być dodany tj. `nav__li nav__li--secondary`